### DISCUSSION TITLE
Comunicação bilíngue | Bilingual communication

### DISCUSSION THREAD

**Discussion by ogecece** [okfn-brasil/querido-diario-comunidade#4] [General]
No repositório de raspadores, a issue okfn-brasil/querido-diario#498, também anteriormente na Serenata na issue okfn-brasil/serenata-de-amor#87 e em outros repositórios onde a maior parte das contribuições é feita por pessoas falantes de língua portuguesa há a discussão sobre o "porquê manter a documentação em inglês?".

Assim, estamos iniciando o movimento de **manter a documentação dos repositórios do Querido Diário bilíngue** (português-inglês) e a **comunicação em issues, PRs, etc. priorizando o português** mas se possível (se souber) também acessível a outros idiomas.

Quem desejar, pode ficar à vontade e contribuir mais ainda com esta discussão aqui :)

---

### English

In the spiders' repo, the issue okfn-brasil/querido-diario#498, also previously at Serenata in the issue okfn-brasil/serenata-de-amor#87 and in other repos where most of the contributions are made by speakers of the Portugues lanagues discussions exists about "why keep the documentation in English?".

Therefore, we are starting the movement of **maintaining the Querido Diário's repos documentations bilingual** (portugues-english) and the **communication in issues, PRs, etc. prioritizing Portuguese** but keeping it accessible to other languages (if possible).

Anyone who desires can make yourself comfortable and contribute even more with this discussion here :)

**Comment by lguima** [comment]
Pros arquivos markdown, pesquisando rapidamente, o padrão que mais encontrei foi: `<nome-do-arquivo><idioma>-<região>.md`.

A proposta da pasta `languages/<idioma>-<região>/<nome-do-arquivo>.md` foi baseada em algum outro projeto?

Pensando aqui, talvez fosse interessante a gente ter o idioma/região no nome do arquivo, não apenas na pasta, simplifica a criação. E aí nesse caso poderia ficar todos na pasta `.github/`.

E para referenciar os idiomas disponíveis, usar links com o nome do idioma apontando para o respectivo arquivo, em vez de "Clique aqui".

Então, se ainda tiver espaço para definirmos um padrão, segue a sugestão de organização de arquivos:

```
.github/README-en-US.md
```

E de link para outros idiomas:
![github-languages](https://user-images.githubusercontent.com/1562097/156270788-c86b352c-1621-4324-83c8-82d3e3564b6c.png)

_Referência_
- https://github.com/markdown-localization/mdlm-sh/tree/master/test/example-localized

---

**English**

In a quick research related to markdown files the most common pattern I found was: `<filename><language>-<region>.md`.

Was the folder suggestion `languages/<language>-<region>/<filename>.md` based on another project?

Thinking about it I found interesting having the language/region in the filename, not only in the folder name. It simplifies the file creation. And on that case all of them could stay in the folder `.github/`.

And to reference the available languages to use the language name linking to the respective file, instead of "Click here".

So, if we still have space to define a pattern, here is a suggestion for the file organization:

```
.github/README-en-US.md
```

And a suggestion for other languages link:
![github-languages](https://user-images.githubusercontent.com/1562097/156270788-c86b352c-1621-4324-83c8-82d3e3564b6c.png)

_Reference_
- https://github.com/markdown-localization/mdlm-sh/tree/master/test/example-localized
